sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/api/Microsoft.Azure.WebJobs.Extensions.EventHubs.netstandard2.0.cs

```diff
@@ -49,6 +49,11 @@ public enum OffsetType
         FromEnd = 1,
         FromEnqueuedTime = 2,
     }
+    public partial class TriggerPartitionContext : Azure.Messaging.EventHubs.Consumer.PartitionContext
```

Comment:
Additional thoughts on naming:

- `EventHubTriggerPartitionContext` :  This is a mouthful, but explicitly identifies what it is and where it appears.

- `TriggerPartitionContext` : This doesn't offer a hint that it is Event Hubs-specific, but it may be enough.  Even were a developer using a non-Event Hubs trigger and referencing this package for the output binding, the association with the `PartitionContext` from the Event Hubs library would differentiate.

- `EventHubPartitionContext` : This doesn't offer a hint of the association with a Functions trigger.  My concern is that it may cause confusion and developers may assume that it is part of the Event Hubs library and not the extension.

- `PartitionContext` : This isn't really an option; even when differentiated by namespace, this would be confusing and difficult to implement in the binding logic.

Reply:
- `HubTriggerPartitionContext` : Potential compromise with a nod to Event Hubs but maybe still too verbose.

Reply:
@KrzysztofCwalina, @tg-msft:  Would you be so kind as to provide the architect perspective around naming for this?